dockers/docker-nat/Dockerfile.j2

```diff
@@ -2,6 +2,8 @@
 FROM docker-config-engine-buster
 
 ARG docker_container_name
+ARG manifest
```

Comment:
Should the J2 file changes be in a different PR, let this PR be just the dhcp-relay changes.

Reply:
These changes are in https://github.com/Azure/sonic-buildimage/pull/5939.
It is not simple to generate a PR which touches changes done in other PR.
So after https://github.com/Azure/sonic-buildimage/pull/5939 is merged this diff will be shown as only DHCL relay changes.